frontend/packages/console-app/src/components/cloud-shell/CloudShellDrawer.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
 import { Tooltip, Flex, FlexItem, FlexModifiers, Button } from '@patternfly/react-core';
 import { CloseIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
-import { Drawer } from '@console/shared';
+import Drawer from '@console/shared/src/components/drawer/Drawer';
 import MinimizeRestoreButton from './MinimizeRestoreButton';
+
 import './CloudShellDrawer.scss';
 
-export type CloudShellDrawerProps = {
-  open: boolean;
+type CloudShellDrawerProps = {
+  open?: boolean;
```

Comment:
Not sure, why this is optional. We don't provide a default value for open also if this is omitted from the props then we would have no way to render the drawer component.